docs/js.md

```diff
@@ -231,66 +253,33 @@ org.scalajs.dom.window.setTimeout(() => {
 }, 3000)
 ```
 
-## Using scalajs-bundler
```

Comment:
what happened to all this text below? should it be part of this change?

Reply:
It was moved to the bottom of the page and preserved in a separate section. 

As far as I can tell, scala-js bundler is no longer actively used with mdoc.